src/ScriptCs.ComponentModel.Composition.Test/ScriptCsCatalogFacts.cs

```diff
@@ -755,6 +754,8 @@ private static Mock<IFileSystem> GetMockFileSystem(string[] fileNames, string[]
             fileSystem.SetupGet(f => f.BinFolder).Returns("scriptcs_bin");
             fileSystem.SetupGet(f => f.DllCacheFolder).Returns(".scriptcs_cache");
             fileSystem.SetupGet(f => f.NugetFile).Returns("scriptcs_nuget.config");
+            fileSystem.SetupGet(f => f.GlobalFolder).Returns(Environment.CurrentDirectory);
```

Comment:
I am not comfortable with using `Environment` in a unit test, is this mandatory?